.github/workflows/publish.yml

```diff
@@ -33,7 +20,19 @@ jobs:
           password: ${{ secrets.RELEASE_DAGGER_CI_TOKEN }}
       - run: ./hack/make-prod dagger:publish ${{ github.ref_name }}
 
-      - name: Bump SDK Engine Dependencies
+      - name: "Publish Engine & CLI"
+        env:
+          GITHUB_TOKEN: ${{ secrets.RELEASE_DAGGER_CI_TOKEN }}
+          DAGGER_ENGINE_IMAGE: ${{ secrets.RELEASE_DAGGER_ENGINE_IMAGE }}
+          AWS_ACCESS_KEY_ID: ${{ secrets.RELEASE_AWS_ACCESS_KEY_ID }}
+          AWS_SECRET_ACCESS_KEY: ${{ secrets.RELEASE_AWS_SECRET_ACCESS_KEY }}
+          AWS_REGION: ${{ secrets.RELEASE_AWS_REGION }}
+          AWS_BUCKET: ${{ secrets.RELEASE_AWS_BUCKET }}
+          ARTEFACTS_FQDN: ${{ secrets.RELEASE_FQDN }}
+          HOMEBREW_TAP_OWNER: ${{ secrets.RELEASE_HOMEBREW_TAP_OWNER }}
+        run: ./hack/make dagger:publish ${{ github.ref_name }}
```

Comment:
Aren’t we double publishing?

Reply:
Fixed in https://github.com/dagger/dagger/pull/4028/files#diff-551d1fcf87f78cc3bc18a7b332a4dc5d8773a512062df881c5aba28a6f5c48d7L21-R32